C++/10.cpp

```diff
@@ -0,0 +1,12 @@
+#include<bits/stdc++.h>
```

Comment:
@harish3600 Please avoid using this bits/stdc++ header file as the compiler has to actually read in and parse every included header (including recursively included headers) every single time that translation unit is compiled.

P.S. Using it in competitive competitions is a good idea but not everywhere and that too because it saves a lot of time.